test/cpp/interop/xds_interop_client.cc

```diff
@@ -470,6 +494,14 @@ void BuildRpcConfigsFromFlags(RpcConfigurationsQueue* rpc_configs_queue) {
     if (metadata_iter != metadata_map.end()) {
       config.metadata = metadata_iter->second;
     }
+    if (absl::GetFlag(FLAGS_request_payload_size) > 0) {
```

Comment:
Please move the if block outside the `for` loop. Inside the for loop, print a warning log message when `request_payload_size` > 0, but an `EmptyCall` is requested.

Reply:
Done